dj/construction/build.py

```diff
@@ -189,12 +189,7 @@ def _build_query_ast(  # pylint: disable=too-many-arguments
     """
     select = query._to_select()  # pylint: disable=W0212
     _build_select_ast(session, select, build_plan, build_plan_depth, database, dialect)
-    for i, exp in enumerate(select.projection):
```

Comment:
I moved this logic to a method on the `ast.Select` class, mainly because it only applies to that class. I also needed to use this in the validate route and figured DRY. (cc: @CircArgs)

Reply:
looks good